### DISCUSSION TITLE
Welcome to pls-cli Discussions!

### DISCUSSION THREAD

**Discussion by guedesfelipe** [guedesfelipe/pls-cli#1] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * ❔Ask questions you’re wondering about.
  * 🎨 **Share themes configs**
  * 💡Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by aravezskinteeth** [comment]
Here is mine with a `catppuccin-mocha` palette:

```shell
export PLS_ERROR_LINE_STYLE="#EBA0AC"
export PLS_ERROR_TEXT_STYLE="#F38BA8 bold"

export PLS_WARNING_LINE_STYLE="#F9E2AF"
export PLS_WARNING_TEXT_STYLE="#F9E2AF bold"

export PLS_UPDATE_LINE_STYLE="#A6E3A1"
export PLS_UPDATE_TEXT_STYLE="#A6E3A1 bold"

export PLS_INSERT_DELETE_LINE_STYLE="#CBA6F7"

export PLS_INSERT_DELETE_TEXT_STYLE="#9399B2"

export PLS_MSG_PENDING_STYLE="#9399B2"
export PLS_TABLE_HEADER_STYLE="#F5C2E7"
export PLS_TASK_DONE_STYLE="#9399B2"
export PLS_TASK_PENDING_STYLE="#CBA6F7"
export PLS_HEADER_GREETINGS_STYLE="#FAB387"
export PLS_QUOTE_STYLE="#9399B2"
export PLS_AUTHOR_STYLE="#9399B2"

export PLS_BACKGROUND_BAR_STYLE="bar.back"
export PLS_COMPLETE_BAR_STYLE="bar.complete"
export PLS_FINISHED_BAR_STYLE="bar.finished"
```
<img width="790" alt="SCR-20230228-ul5" src="https://user-images.githubusercontent.com/104392087/222034259-da61d5c8-4814-471d-b65d-7069961c437a.png">

**Reply by guedesfelipe** [reply]
Awesome!! I'll put it as a theme in the documentation.

Thanks for sharing

Welcome to the community  😎

**Reply by guedesfelipe** [reply]
https://guedesfelipe.github.io/pls-cli/themes/#catppuccin-mocha
